thenewboston_node/blockchain/tests/fixtures.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+
+from thenewboston_node.business_logic.models import Block, NodeDeclarationSignedChangeRequest
+from thenewboston_node.business_logic.node import get_node_signing_key
+
+
+@pytest.fixture
+def declared_node_file_blockchain(forced_file_blockchain):
```

Comment:
the fixture should live here: thenewboston_node/business_logic/tests/fixtures/blockchain.py